src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java

```diff
@@ -279,7 +270,7 @@ private Node createListEntryForProject(final Project p) {
          alert.setHeaderText("Delete project '" + p.getName() + "'.");
          alert.setContentText(
                "The project will just be hidden from display, as there may be work references to this project.");
-
+         alert.initOwner(mainStage);
```

Comment:
for some popups we have an owner and for some not. can we make it consistent?
I think setting the owner is correct - and then overwrite the icon